gemac/management.py

```diff
@@ -37,12 +36,12 @@ def management(hostintf, mdiointf, reset):
     Args:
         hostintf - Instance of 'HostManagementInterface' class in interfaces.
         mdiointf - Instance of 'MDIOInterface' class in interfaces.
+        configregs - List of 10 32-bits wide Configuration Registers.
+        addrtable - List of 4 48-bits wide MAC Addresses to be used by
+            Address Filter.
         reset: Asynchronous reset Signal from Host
```

Comment:
In myhdl a design uses `ResetSignal` and the system designer (the one building a system out of a bunch of blocks) can determine what the global reset should look like, in the comment specify system reset or list the reason whey it needs to be a specific reset.